hardhat.config.js.fork

```diff
@@ -0,0 +1,73 @@
+/*global process*/
+
+require("hardhat-deploy");
+require("hardhat-deploy-ethers");
+require("@nomiclabs/hardhat-waffle");
+require("solidity-coverage");
+require("hardhat-gas-reporter");
+require("hardhat-contract-sizer");
+require("@nomiclabs/hardhat-etherscan");
+
+const accounts = {
+    mnemonic: "test test test test test test test test test test test junk",
+    accountsBalance: "100000000000000000000000000",
+};
+
+const ALCHEMY_API_KEY = process.env.ALCHEMY_API_KEY;
+let GOERLI_MNEMONIC = process.env.GOERLI_MNEMONIC;
+if (!GOERLI_MNEMONIC) {
+    GOERLI_MNEMONIC = accounts.mnemonic;
+}
+const ETHERSCAN_API_KEY = process.env.ETHERSCAN_API_KEY;
+
+module.exports = {
+    networks: {
+        local: {
+            url: "http://localhost:8545",
+        },
+        mainnet: {
+            url: "https://eth-mainnet.g.alchemy.com/v2/" + ALCHEMY_API_KEY,
+            chainId: 1,
+        },
+        goerli: {
+            url: "https://eth-goerli.alchemyapi.io/v2/" + ALCHEMY_API_KEY,
+            chainId: 5,
+            accounts: {
+                mnemonic: GOERLI_MNEMONIC,
+                path: "m/44'/60'/0'/0",
+                initialIndex: 0,
+                count: 20,
+                passphrase: "",
+            },
+        },
+        hardhat: {
+            allowUnlimitedContractSize: true,
+            accounts,
+            forking: {
+                url: "https://eth-mainnet.g.alchemy.com/v2/" + ALCHEMY_API_KEY,
+                # The block number of the original Curve Voting Escrow deployment
```

Comment:
```suggestion
                // The block number of the original Curve Voting Escrow deployment
```

Reply:
One of the motivations that bring my compiler breaking when running `npx run fork`

Reply:
But I have seen that you on the new branch `wveOLAS2` you updated it, and there it correctly run

Reply:
Something has to do with hardhat update and scripts.